apps/admin/src/app/pages/user/add-edit-user.tsx

```diff
@@ -56,16 +56,20 @@ const AddEditUser = () => {
 
     return (
         <Page title={`${userId ? 'Edit User' : 'Add User'}`}>
-            <Container maxWidth={false}>
+            <Container maxWidth='lg'>
                 <CustomBreadcrumbs
-                    heading="Users"
+                    heading="Edit User"
```

Comment:
If you are using same page then when adding title should be `Add User`

Reply:
fix name